docs/monitoring.md

```diff
@@ -32,3 +34,43 @@ Limitations:
 - An index pattern `logstash-*` may need to be create on the first access
 - The ElasticSearch server is not clustered and has limited querying capacity. It can be easily overloaded.
 - Logs are kept for two weeks.
+
+
+## IDR FTP Monitoring
+
+The IDR FTP server has limited Prometheus monitoring running on the same server with alerts for low available disk space.
+
+
+## IDR Pilot Monitoring
+
+All IDR pilot test instances are currently monitored by a single monitoring server.
+This is accessed using the same commands as above, using `idr-pilot` as the proxy.
+
+
+## Deployment
```

Comment:
Looking at the outline (i.e. `##` markdown level) you're trying to say here, "this is for the people setting it up" and the bits above are more for the "users"?

Reply:
Yes, I've re-arranged the doc slightly, is that clearer?